.ci/scripts/generate-build-tree

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# e - non-interactive terminals will exit on failures immediately
+# x - print commands before exiting
+set -ex
```

Comment:
Is this actually needed for a single-command shell script?

Reply:
I removed the `-e` and polished the comment.